benchmarks/haskell/Benchmarks/Micro.hs

```diff
@@ -0,0 +1,22 @@
+-- | Benchmarks on artificial data. 
+
+module Benchmarks.Micro (benchmark) where
+
+import qualified Data.Text.Lazy as TL
+import qualified Data.Text as T
+import Test.Tasty.Bench (Benchmark, bgroup, bench, nf)
+
+benchmark :: Benchmark
+benchmark = bgroup "Micro"
+  [ -- Accessing i-th element should take O(i) time.
+    -- The 2k case should run in 2x the time of the 1k case.
```

Comment:
Sounds like a use case for [`bcompareWithin`](https://hackage.haskell.org/package/tasty-bench-0.3.5/docs/Test-Tasty-Bench.html#v:bcompareWithin)!

Reply:
Fancy!